clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1271,4 +1274,34 @@ private static byte checkRange(final byte i) {
         }
         return map;
     }
+
+    /**
+     * Convert an ISO8601 based timestamp to an epoch value
```

Comment:
nit: I don't know from the top of my head what the standard dictates. Might be worth adding to the comment?

Reply:
Hi @mjsax thanks for reviewing my PR.

I removed "ISO8601" that from the comment as it is Defined in very Dateformatter. please let me know ur thoughts on that.